### ISSUE TITLE
Veinminer Mining Stone + Fix

### ISSUE THREAD

**Issue by chimeralchimera** [scalda/FoolCraft#60]
I'm not really sure if this is intentional or not, but I was having an issue where Veinminer was mining regular stone, instantly using up an entire pick when doing so. Now, I've played UHCs for years now, and as a result, have a tendency to sneak-mine constantly, and as you might expect, I kept accidentally breaking my picks!  I also didn't want to switch to having it work by pressing another key, as I rather like the idea of sneak-mining ore veins, just not stone. Since stone isn't on the list of things Veinminer's supposed to mine anyway, I decided to dig around and figure out what the problem was.

So, long story short, I ended up digging into the config file ("C:\<curse directory path>\Minecraft\Instances\FoolCraft\config\veinminer\general.cfg") and found that the overrides at the bottom were both set to true.  Now, I haven't had the chance to extensively test this yet, but setting these to false seems to have stopped the issue. 

At any rate, I wasn't sure if I should post this here as an issue or not, but I thought I might as well. I'm also wondering, were those overrides supposed to be set to true for some reason, or was this just an oversight?

**Comment by Omegapsychosis** [comment]
I believe these options were enabled on purpose as Iskall85 didn't want the grindy feeling many packs have. It's meant to be fun, trolly and semi-OP.

**Comment by scalda** [comment]
What omOmegapsychosis said
